lib/bolt/target.rb

```diff
@@ -129,6 +129,11 @@ def transport
       inventory_target.transport
     end
 
+    def transport_config
+      inventory_target.transport_config.to_h
+    end
+    alias options transport_config
```

Comment:
We only need the `options` method because all the transports use it, right? ie. the only consumers of it are within Bolt?

Reply:
Correct